sass/main.scss

```diff
@@ -168,6 +168,37 @@ footer {
   }
 }
 
+.on-demand {
+  background: url("/images/bg-banner-1.jpg") center center no-repeat;
```

Comment:
wasn't there something like {{url }} in the CSS to avoid hardcoding paths?

Reply:
I think that was in the HTML not the css